core/api/network/setup_net.hpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <libp2p/host/host.hpp>
+#include "api/network/network_api.hpp"
+
+namespace fc::api {
+  using libp2p::Host;
+
+  inline void fillNetApi(const std::shared_ptr<NetworkApi> &api,
+                         const PeerInfo &api_peer_info,
+                         const std::shared_ptr<libp2p::Host> &host,
```

Comment:
Either use Host or remove `using libp2p::Host;` 
```suggestion
                         const std::shared_ptr<Host> &host,
```